cmd/auth/key/delete.go

```diff
@@ -42,19 +42,19 @@ var deleteCmd = &cobra.Command{
 			}
 		}
 
-		var merr = new(multierror.Error)
+		var merr = multierror.NewPrefixed("multiple delete")
```

Comment:
I think this way follows convention with the other errors 
```suggestion
		var merr = multierror.NewPrefixed("delete key")
```